tias/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "tias",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.2",
+    "@testing-library/react": "^12.1.3",
+    "@testing-library/user-event": "^13.5.0",
+    "@types/jest": "^27.4.1",
+    "@types/node": "^16.11.25",
+    "@types/react-dom": "^17.0.11",
```

Comment:
Consider these as `devDependencies`? Do we need to package them with the release?